source/mainnet/smart-contracts/guides/quick-start.rst

```diff
@@ -0,0 +1,169 @@
+.. include:: ../../variables.rst
+.. _sc-quick-start:
+
+============================================
+Concordium smart contracts quick start guide
+============================================
+
+This guide is intended to be used by experienced developers who want to get up and running quickly with a smart contract deployed on chain with which they can interact.
+
+.. dropdown:: Step 1 - Set up developer environment
+
+    Download and install the following tools in this order:
+
+    #. `rustup`_
+    #. :ref:`cargo-concordium<cargo-concordium-testnet>`
+    #. :ref:`concordium-client<concordium-node-and-client-download-testnet>`
+    #. :ref:`a node<concordium-node-and-client-download-testnet>` for deploying and interacting
+
+    For more detailed information, see :ref:`Install tools for development<setup-tools>`.
+
+.. dropdown:: Step 2 - Set up smart contract project
+
+    You can choose to start a smart contract project from a `template <https://github.com/Concordium/concordium-rust-smart-contracts/tree/main/templates>`_ or from scratch.
+
+    **From a template**
+
+    To generate smart contracts from the templates, the ``cargo-generate`` crate is required. To install ``cargo-generate`` run the following command:
+
+    .. code-block:: console
+
+        $cargo install --locked cargo-generate
+
+    To start a new Concordium smart contract project from a template, run the command:
+
+    .. code-block:: console
+
+        $cargo concordium init
+
+    The path where the project should be created can be provided with the ``--path`` option.
+
+    You can use the `concordium-std`_ library to help you build smart contracts or build without Rust's ``std``. To add the library, open ``Cargo.toml`` and add the line
+    ``concordium-std = "*"`` (preferably, replace the `*` with the latest version of `concordium-std`_) in the ``[dependencies]`` section:
+
+        [dependencies]
+        concordium-std = "3.0"
+
+    The crate documentation is on docs.rs_.
```

Comment:
Perhaps link directly to the crate `https://docs.rs/concordium-std/latest/concordium_std/`

Reply:
Also above it is referred to as "library" and here as "crate". It would be better if either option is used consistently. I recommend "library" since it is more widely known.